.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
You probably want to remove this.

Reply:
This is for development using Intellij tools, which creates ".idea" dir for storing configuration in the root repo folder.